framework/src/meshgenerators/TiledMeshGenerator.C

```diff
@@ -76,6 +71,23 @@ TiledMeshGenerator::generate()
   std::unique_ptr<MeshBase> initial_mesh = std::move(_input);
   std::unique_ptr<ReplicatedMesh> mesh = dynamic_pointer_cast<ReplicatedMesh>(initial_mesh);
 
+  // Getting the x,y,z widths
+  std::set<subdomain_id_type> sub_ids;
+  mesh->subdomain_ids(sub_ids);
+  BoundingBox bbox(Point(0, 0, 0), Point(0, 0, 0));
```

Comment:
Defaulting to zero can fail if your mesh is completely in positive or completely in negative coordinates. You might want to start with a max and min that are set to the "min" and "max" respectively.